internal/network/command_private_link_attachment_connection.go

```diff
@@ -73,6 +78,9 @@ func printPrivateLinkAttachmentConnectionTable(cmd *cobra.Command, connection ne
 		case connection.Status.Cloud.NetworkingV1AzurePrivateLinkAttachmentConnectionStatus != nil:
 			out.AzurePrivateLinkServiceAlias = connection.Status.Cloud.NetworkingV1AzurePrivateLinkAttachmentConnectionStatus.GetPrivateLinkServiceAlias()
 			out.AzurePrivateLinkServiceId = connection.Status.Cloud.NetworkingV1AzurePrivateLinkAttachmentConnectionStatus.GetPrivateLinkServiceResourceId()
+		case connection.Status.Cloud.NetworkingV1GcpPrivateLinkAttachmentConnectionStatus != nil:
+			out.GcpServiceAttachmentId = connection.Status.Cloud.NetworkingV1GcpPrivateLinkAttachmentConnectionStatus.GetPrivateServiceConnectServiceAttachment()
+			out.GcpPrivateServiceConnectConnectionId = connection.Status.Cloud.NetworkingV1GcpPrivateLinkAttachmentConnectionStatus.GetPrivateServiceConnectConnectionId()
```

Comment:
It looks this is being set twice, on line 70 and here.

Is `GetPrivateServiceConnectConnectionId` part of the response for both Spec and Status?